distribution/src/resources/config-tool/templates/conf/axis2/axis2_client.xml.j2

```diff
@@ -0,0 +1,298 @@
+<!--
```

Comment:
Shall we update the license headers?

Reply:
This is template file taken from the original axis2_client.